app/views/works/_s3_resources.html.erb

```diff
@@ -14,29 +16,32 @@
           <tbody>
             <% if @work.uploads.empty? %>
               <tr class="files">
-                <td><span><%= t('works.form.curation_uploads.empty') %></span></td>
+                <td><span>No files in S3</span></td>
+                <td><span></span></td>
                 <td><span></span></td>
               </tr>
-              <% end %>
-            </tr>
+            <% end %>
           </tbody>
           <tfoot></tfoot>
         </table>
-        <!-- End of file download table -->
       </div>
     </div>
   </div>
 </section>
 
 <script type="text/javascript">
   $(function() {
+    // This is needed so that we can swap the URL during testing
+    // (for more info see external_identifier_specs.rb)
+    var fileListUrl = '<%= Rails.env.test? ? WORKS_FILE_LIST_URL.gsub(/place-holder/, @work.id.to_s) : work_file_list_path(@work) %>';
```

Comment:
For the other URLS, we set it in both cases (test or not).  Then there is no need for the logic here...  See the ORCID for and [example](https://github.com/pulibrary/pdc_describe/search?q=ORCID_URL)

Reply:
Adding a reference to `work_file_list_path(id: "place-holder")` in `./config/environments/development.rb`  results in error "undefined method `work_file_list_path' for main:Object (NoMethodError)" -- I don't think all the Rails configuration is available at that stage.